pkg/csi/service/wcp/controller.go

```diff
@@ -618,7 +623,6 @@ func (c *controller) createBlockVolume(ctx context.Context, req *csi.CreateVolum
 		VsanDirectDatastoreURL:  selectedDatastoreURL,
```

Comment:
We should change this variable name to Selected hostlocal datastore as we are going to use this value for vSAN SNA as well as VSAN Direct datastore